spec/rubocop/cop/rspec/multiple_describes_spec.rb

```diff
@@ -50,4 +50,139 @@
       end
     RUBY
   end
+
+  context 'when EnforcedStyle is :splitting' do
+    let(:cop_config) { { 'EnforcedStyle' => :splitting } }
+
+    it 'check first describe in nested describe' do
+      expect_offense(<<-RUBY)
+        RSpec.describe MyClass do
+          describe '#do_something' do
+          ^^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+            subject { my_class_instance.foo }
+          end
+        end
+      RUBY
+    end
+
+    it 'check first describe in each nested describe' do
+      expect_offense(<<-RUBY)
+        describe MyClassController, type: :controller do
+          describe 'do_something' do
+          ^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+            describe 'do_something_in_describe' do; end
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use nested describe - try to split them.
+          end
+          describe 'do_something_else' do; end
+        end
+      RUBY
+    end
+
+    it 'ignores describe in top level groups' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass, '#do_something' do
+          subject { my_class_instance.foo }
+        end
+        RSpec.describe MyClass, '#do_something_else' do
+          subject { my_class_instance.bar }
+        end
+      RUBY
+    end
+
+    it 'ignores describe when rspec matcher exists' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          RSpec::Matchers.define :be_a_multiple_of do |expected|
+            match do |actual|
+              actual % expected == 0
+            end
+          end
+
+          describe '#do_something' do
+            subject { my_class_instance.foo }
+          end
+
+          describe '#do_something_else' do
+            subject { my_class_instance.bar }
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores describe when shared example group exists' do
+      expect_no_offenses(<<-RUBY)
+        RSpec.describe MyClass do
+          shared_examples 'behaves' do;end
+
+          describe '#do_something' do
+            subject { my_class_instance.foo }
+          end
+
+          describe '#do_something_else' do
+            subject { my_class_instance.bar }
+          end
+        end
+      RUBY
+    end
+
+    it 'ignores describe when shared example for group exists' do
```

Comment:
`shared_examples_for` is an alias, this example is redundant.

Reply:
I understand. I deleted the example.